chemprop/train/make_predictions.py

```diff
@@ -31,16 +30,25 @@ def make_predictions(args: PredictArgs, smiles: List[List[str]] = None) -> List[
     update_prediction_args(predict_args=args, train_args=train_args)
     args: Union[PredictArgs, TrainArgs]
 
-    if args.atom_descriptors == 'feature':
-        set_extra_atom_fdim(train_args.atom_features_size)
-
-    if args.bond_features_path is not None:
-        set_extra_bond_fdim(train_args.bond_features_size)
+    # Load model and scalers
+    models=(load_checkpoint(checkpoint_path, device=args.device) for checkpoint_path in args.checkpoint_paths)
+    scalers=(load_scalers(checkpoint_path) for checkpoint_path in args.checkpoint_paths)
+    if not generator:
+        models=list(models)
+        scalers=list(scalers)
```

Comment:
Could you add spaces around the assignment `=` signs? (No spaces around keyword argument `=`)

Reply:
Done!